mmdet/datasets/pipelines/formating.py

```diff
@@ -191,10 +191,15 @@ class DefaultFormatBundle:
     Args:
         img_to_float (bool): Whether to force the image to be converted to
             float type. Default: True.
+        pad_val (dict): A dict for padding value when collcate in parallel,
+            the default value is `dict(img=0, masks=0, seg=255)`.
     """
 
-    def __init__(self, img_to_float=True):
+    def __init__(self,
+                 img_to_float=True,
+                 pad_val=dict(img=0, masks=0, seg=255)):
```

Comment:
Please add unit test

Reply:
We have not filled the data with pad_val in DefaultFormatBundle. So it is hard to add the unit test here.